src/plugins/intel_cpu/src/graph.h

```diff
@@ -263,11 +257,10 @@ class Graph {
     std::vector<NodePtr> constantGraphNodes;
     std::vector<NodePtr> executableGraphNodes;
 
-    MultiCachePtr rtParamsCache;
-    std::shared_ptr<std::mutex> sharedMutex = nullptr;
-    DnnlScratchPadPtr rtScratchPad;
     std::unordered_map<Node*, size_t> syncNodesInds;
 
+    RuntimeEnv::Ptr rtEnv;
+
     void EnforceBF16();
     void setMinSparseRate(float minSparseRate);
```

Comment:
Please remove get rid of this method as well as soon as required data is available inside Runtime Env